server_development/topics/auth-spi.adoc

```diff
@@ -1,7 +1,7 @@
 [[_auth_spi]]
 == Authentication SPI
 
-{project_name} comes out of the box with a bunch of different authentication mechanisms: kerberos, password, and otp.
+{project_name} comes out of the box with a bunch of different authentication mechanisms: kerberos, password, otp and others.
```

Comment:
change "comes out of the box with a bunch of" to "includes a range of"

Reply:
Fixed